pkg/controller/daemon/controller_test.go

```diff
@@ -158,7 +158,7 @@ func addPods(podStore cache.Store, nodeName string, label map[string]string, num
 }
 
 func newTestController() (*DaemonSetsController, *FakePodControl) {
-	client := client.NewOrDie(&client.Config{Host: "", Version: testapi.Experimental.Version()})
+	client := client.NewOrDie(&client.Config{Host: "", Version: testapi.Default.GroupAndVersion()})
```

Comment:
OK

On Mon, Sep 21, 2015 at 3:27 PM, Chao Xu <notifications@github.com> wrote:

> In pkg/controller/daemon/controller_test.go
> <https://github.com/kubernetes/kubernetes/pull/14156#discussion_r40034732>
> :
>
> > @@ -158,7 +158,7 @@ func addPods(podStore cache.Store, nodeName string, label map[string]string, num
> >  }
> >
> >  func newTestController() (*DaemonSetsController, *FakePodControl) {
> > -	client := client.NewOrDie(&client.Config{Host: "", Version: testapi.Experimental.Version()})
> > +	client := client.NewOrDie(&client.Config{Host: "", Version: testapi.Default.GroupAndVersion()})
>
> The client here contains both v1 client and experimental client (
> https://github.com/kubernetes/kubernetes/blob/master/pkg/client/unversioned/client.go#L125
> )
> the Version here is used to config the version of the v1 client. And
> deeper in this NewOrDie function, it will modify the config to set the
> default version for experimental client. (
> https://github.com/kubernetes/kubernetes/pull/14156/files#diff-ee5f86f66628c16b5c04e3a653c118efR138
> )
>
> Currently this client.NewOrDie does not support a way to specify the
> version of the experimental client. It doesn't really hurt so far because
> we only have one version for the experimental group, and we can always
> default to that. IMO, we should fix it in another PR.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/14156/files#r40034732>.
>